### DISCUSSION TITLE
Provide Icon import in documentation

### DISCUSSION THREAD

**Discussion by tommywong-dev** [mantinedev/mantine#956] [Feedback]
There is no clear documentation telling where did Mantine get `<GearIcon/>` in [ActionIcon](https://mantine.dev/core/action-icon/) and [ThemeIcon](https://mantine.dev/core/theme-icon/).

Some people found out it's using `@radix-ui/react-icons` and can be found in discussions but personally I think Mantine should specify it.

**Comment by appurist** [comment]
I've been trying to find out what other icons are available, and where `ActionIcon` and `ThemeIcon` came from, and if they were just examples or literally the only two available. Fortunately @tommywong-dev mentioned `radix-ui` above, or I'd still be lost. Why isn't this mentioned, at least with `ActionIcon` and `ThemeIcon`? One sentence summarizing icons available (or not) is all that is needed. Maybe a link to radix-ui.

**Reply by rtivital** [reply]
Both ThemeIcon and ActionIcon docs are using tabler-icons-react. Full source code is included for all demos, there is not any point in providing anything else.
